examples/embedded_c/embedded_c.c

```diff
@@ -952,8 +956,10 @@ int
 main(int argc, char *argv[])
 {
 	const char *options[] = {
+#if !defined(NO_FILES)
```

Comment:
`!defined(NO_FILESYSTEMS)` or `!defined(NO_FILES)`

Reply:
NO_FILES seems ok. NO_FILESYSTEMS implicitly means NO_FILES. 
Maybe there should be some consistency check of these two defines somewhere (not here).